.github/workflows/build.yml

```diff
@@ -104,6 +104,51 @@ jobs:
         run: |
           docker tag discourse/base:release${{ steps.arch-helper.outputs.arch_postfix_dash }} discourse/base:aarch64
           docker push discourse/base:aarch64
+  base_bookworm:
```

Comment:
This is a simplified version of the [base](https://github.com/discourse/discourse_docker/pull/823/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L17) job.